packages/react/src/publication/useExplorePublications.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  PublicationSortCriteria,
+  PublicationTypes,
+  useExplorePublicationsQuery,
+} from '@lens-protocol/api-bindings';
+
+import { PaginatedArgs, PaginatedReadResult, usePaginatedReadResult } from '../helpers';
+import { useSharedDependencies } from '../shared';
+import { DEFAULT_PAGINATED_QUERY_LIMIT, DEFAULT_PUBLICATION_SORT_CRITERIA } from '../utils';
+import { Publication } from './types';
+
+type UseExplorePublicationsArgs = PaginatedArgs<{
+  observerId?: string;
+  sortCriteria?: PublicationSortCriteria;
+  timestamp?: number;
+  noRandomize?: boolean;
+  publicationTypes?: Array<PublicationTypes>;
+  excludeProfileIds?: Array<string>;
+}>;
+
+export function useExplorePublications({
+  observerId,
+  limit = DEFAULT_PAGINATED_QUERY_LIMIT,
+  sortCriteria = DEFAULT_PUBLICATION_SORT_CRITERIA,
```

Comment:
Let's inline `PublicationSortCriteria.Latest` for now. When we will need to share this we can then think about a proper const. The unit test seems to not explore with/without default params... not that I am advocating it. Maybe for the unit test you can use a value for `PublicationSortCriteria` that is not `PublicationSortCriteria.Latest` and by configuring the query mock it will implicitly check it passes to the query variables.

Reply:
I added a test with custom parameters